beetsplug/bpd/__init__.py

```diff
@@ -640,8 +640,8 @@ class Command(object):
     """A command issued by the client for processing by the server.
     """
 
-    command_re = re.compile(br'^([^ \t]+)[ \t]*')
-    arg_re = re.compile(br'"((?:\\"|[^"])+)"|([^ \t"]+)')
+    command_re = re.compile(r'^([^ \t]+)[ \t]*')
+    arg_re = re.compile(r'"((?:\\"|[^"])+)"|([^ \t"]+)')
```

Comment:
I realize I just don't get what you're trying to say here.  I'll try to clarify more specifically with an example from the bpd tests

```python
    def test_no_args(self):
        s = r'command'
        c = bpd.Command(s)
        self.assertEqual(c.name, u'command')
        self.assertEqual(c.args, [])
```